scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -136,9 +138,9 @@ workflow GvsExtractCallset {
                                                                   else 500])
 
   Int effective_extract_memory_gib = if defined(extract_memory_override_gib) then select_first([extract_memory_override_gib])
-                                     else if effective_scatter_count <= 100 then 40
-                                          else if effective_scatter_count <= 500 then 20
-                                               else 12
+                                     else if effective_scatter_count <= 100 then 37 + extract_overhead_memory_override_gib
+                                          else if effective_scatter_count <= 500 then 17 + extract_overhead_memory_override_gib
+                                               else 9 + extract_overhead_memory_override_gib
```

Comment:
can you document why we want these non-round numbers?

Reply:
OK. By default they sum to the same round numbers they do today since `extract_overhead_memory_override_gib` defaults to 3, but for troublesome "small callsets" like AC AF this variable can be bumped up.